src/styles/main.scss

```diff
@@ -1,3 +1,68 @@
+$day-start: (
+  "mon": 0,
+  "tue": 101px,
```

Comment:
The pixel values for the start day are hardcoded. It is better to try to use flex technology and not hardcode these values.